src/apps/service/apps/de/listings.clj

```diff
@@ -315,6 +315,13 @@
                   :value (workspace-metadata-beta-value)}]
     (set (metadata-client/filter-by-avus username app-ids [beta-avu]))))
 
+(defn- filter-app-ids-by-community
+  "Filters the given list of app-ids into a set containing the ids of apps marked as `beta`"
```

Comment:
It looks like this docstring hasn't been updated.

Reply:
Oops, clipboard inheritance 😖
Good catch!